### PR TITLE
Improve color parsing to generate contrast color

### LINKED ISSUE #156: Text is white over white background for some Details pages
Text is white over white background for some Details pages, for example:
https://pwa-directory.appspot.com/pwas/5664902681198592

### PR DESCRIPTION
**Pull request by andreban**
- Added 'parse-color' package.
- Added failing '#fff' color to tests.

Closes #156

### CODE REVIEW

File: views/helpers/index.js [hunk 1]
```diff
@@ -15,18 +15,22 @@
...
-    hexcolor = hexcolor.substr(1, hexcolor.length);
+  // Assume that a 6 digit string is a color.
+  if (hexcolor.length === 6 && hexcolor[0] !== '#') {
+    hexcolor = '#' + hexcolor;
   }
-  const r = parseInt(hexcolor.substr(0, 2), 16);
-  const g = parseInt(hexcolor.substr(2, 2), 16);
-  const b = parseInt(hexcolor.substr(4, 2), 16);
+
+  const parsedColor = parseColor(hexcolor);
```
Comment (ithinkihaveacat) [inline views/helpers/index.js]: L30-L33 could also be written as

```js
const [r, g, b] = parseColor(hexcolor).rgb;
```

via [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Basic_variable_assignment).

Comment (andreban) [inline views/helpers/index.js]: Didn't know about this one. Changed the code. Thanks, @ithinkihaveacat

**Comment by juliantoledo** [comment]
@andreban It's failing the Node4 tests at travis.

File: views/helpers/index.js (hunk 1 above)
Comment (juliantoledo) [inline views/helpers/index.js]: It's that Node4 compliant?

**Comment by andreban** [comment]
It seems Node4 doesn't support the destructuring assignment syntax. Will revert the change.

**Comment by juliantoledo** [comment]
@andreban it works :)

LGTM (just fix /Users/jtoledo/GitHub/gulliver/views/helpers/index.js
  34:1  error  Trailing spaces not allowed  no-trailing-spaces)

**Pull request merged**